.github/workflows/publish.yml

```diff
@@ -152,12 +152,12 @@ jobs:
       - name: "Setup Rust"
         uses: dtolnay/rust-toolchain@stable
 
-      - name: "ALWAYS print engine logs - especially useful on failure"
-        if: always()
+      - name: "Print Engine logs on failure"
```

Comment:
IMO, we should keep these to always if we can.

I've found it really useful to go back through old successful runs and compare logs to new failing runs.

Reply:
OK, will change this before merging.